### PR TITLE
Add docs for pnpm and yarn 2+

### PR DESCRIPTION
**Pull request by wyzlle**
Updated docs for node provider.

### CODE REVIEW

File: docs/node.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # Node Support
 
-For both NPM and Yarn providers, the following environment variables are set.
+For all three NPM, Yarn and PNPM providers, the following environment variables are set.
```
Comment (Milo123459) [inline docs/node.md:3]:
```suggestion
The NPM, Yarn and PNPM providers all have the following environment variables set:
```

File: docs/node.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
 - `NODE_ENV=production`
 - `NPM_CONFIG_PRODUCTION=false`: Ensure that dev deps are always installed
 
 ## [NPM](https://www.npmjs.com/)
 
 **Install**:
-
+If lockfile is found in source code
```
Comment (Milo123459) [inline docs/node.md:11]:
```suggestion
If a lockfile is found in the source code
```

File: docs/node.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+```
+npm ci
+```
+If lockfile isn't found in source code
```
Comment (Milo123459) [inline docs/node.md:15]:
````suggestion
If a lockfile isn't found in the source code
```

File: docs/node.md [hunk 4]
```diff
@@ -26,7 +30,7 @@ npm run build
 Start script found in `package.json`
 
 ```
-npm run start
+npm start
```
Comment (Milo123459) [inline docs/node.md:33]: Should be reverted as your PR changing this was closed.
```suggestion
npm run start
```

**Review by Milo123459 — APPROVED** [review]

File: docs/node.md [hunk 5]
```diff
@@ -45,28 +48,68 @@ node index.js
 
 Yarn is detected if a `yarn.lock` file is found at the root level.
 
-_Currently only Yarn 1 is supported_
-
 **Install**:
 
+For Yarn 1
```
Comment (devtomio) [inline docs/node.md:53]:
```suggestion
For [Yarn 1](https://classic.yarnpkg.com/lang/en/)
```

File: docs/node.md [hunk 6]
```diff
@@ -45,28 +48,68 @@ node index.js
...
 ```
 yarn install --frozen-lockfile
 ```
 
+For Yarn 2+
```
Comment (devtomio) [inline docs/node.md:58]:
```suggestion
For [Yarn 2+](https://yarnpkg.com/)
```

**Review by devtomio — CHANGES_REQUESTED** [review]

**Pull request merged**